bundles/org.openhab.core.automation/src/main/java/org/openhab/core/automation/RuleManager.java

```diff
@@ -92,4 +95,13 @@
      * @param context the context that is passed to the conditions and the actions of the rule.
      */
     void runNow(String uid, boolean considerConditions, @Nullable Map<String, Object> context);
+
+    /**
+     * Simulates the execution of all rules with tag 'Schedule' for the given time interval.
+     * The result is sorted ascending by execution time.
+     *
+     * @param untilDate {@link Date} until the executions should be simulated.
```

Comment:
Can you add JavaDoc for both parameters and rename this one to `until`:

```suggestion
     * @param until {@link ZonedDateTime} until the executions should be simulated.
```

Reply:
I've renamed all from untilDate to until and added missing documentation.